test/helpers/pluralize.test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const assert = require('assert');
+const pluralize = require('../../lib/helpers/pluralize');
+
+// David Xie: test pluralization rules (re: gh-11515)
+// https://github.com/Automattic/mongoose/issues/11515
+
+// six => sixes
+// sex => sexes
+// vertex => vertices
+// Index => indices (David Xie's fix)
+describe('pluralize.js testings', function() {
+  it('Pluralize words', function(done) {
+    const original = ['six', 'sex', 'vertex', 'Index'];
+    const plurals = original.map((element) => pluralize(element));
+    const expected = ['sixes', 'sexes', 'vertices', 'indices'];
+    assert.deepStrictEqual(plurals, expected);
+    done();
```

Comment:
We don't need to have a `done();` here anymore, since mocha now runs synchronous tests to the end and will throw an error if the assertion fails.